modules/persistent-storage-csi-gcp-filestore-wif.adoc

```diff
@@ -0,0 +1,126 @@
+  name: openshift-gcp-filestore-csi-driver-operator
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  serviceAccountNames:
+  - gcp-filestore-csi-driver-operator
+  - gcp-filestore-csi-driver-controller-sa
+  secretRef:
+    name: gcp-filestore-cloud-credentials
+    namespace: openshift-cluster-csi-drivers
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+	kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/file.editor
+    - roles/resourcemanager.tagUser
+    skipServiceCheck: true
+----
+
+.. Use the `CredentialsRequest` file to create a {gcp-short} service account by running the following command:
++
+[source, terminal]
+----
+$ ./ccoctl gcp create-service-accounts --name=<filestore-service-account> <1>
+--workload-identity-pool=<workload-identity-pool> <2> 
+--workload-identity-provider=<workload-identity-provider> <3> 
+--project=<project-id> <4> 
+--credentials-requests-dir=/tmp/credreq <5>
+----
+<1> <filestore-service-account> is a user-chosen name. 
+<2> <workload-identity-pool> comes from Step 2 above.
+<3> <workload-identity-provider> comes from Step 2 above.
+<4> <project-id> comes from Step 1.a above.
+<5> The name of directory where the `CredentialsRequest` file resides.
++
+.Example output
+[source, terminal]
+----
+2025/02/10 17:47:39 Credentials loaded from gcloud CLI defaults
+2025/02/10 17:47:42 IAM service account filestore-service-account-openshift-gcp-filestore-csi-driver-operator created
+2025/02/10 17:47:44 Unable to add predefined roles to IAM service account, retrying...
+2025/02/10 17:47:59 Updated policy bindings for IAM service account filestore-service-account-openshift-gcp-filestore-csi-driver-operator
+2025/02/10 17:47:59 Saved credentials configuration to: /tmp/install-dir/ <1>
+openshift-cluster-csi-drivers-gcp-filestore-cloud-credentials-credentials.yaml
+----
+<1> The current directory.
+
+.. Find the service account email of the newly created service account by running the following command:
++
+[source, terminal]
+----
+cat /tmp/install-dir/manifests/openshift-cluster-csi-drivers-gcp-filestore-cloud-credentials-credentials.yaml | yq '.data["service_account.json"]' | base64 -d | jq '.service_account_impersonation_url'
+----
++
+.Example output
+[source, terminal]
+----
+https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/filestore-se-openshift-g-ch8cm@openshift-gce-devel.iam.gserviceaccount.com:generateAccessToken
+----
++
+In this example output, the service account email is: filestore-se-openshift-g-ch8cm@openshift-gce-devel.iam.gserviceaccount.com.
+
+.Results
+
+You now have the following parameters that you need to install the {gcp-short} Filestore CSI Driver Operator:
+
+* Project number - from Step 1.b
+
+* Pool ID - from Step 2
+
+* Provider ID - from Step 2
+
+* Service account email - from Step 3.c
```

Comment:
Would remove this section, as it's pretty straightforward. If it's really needed, I would at least remove the specific step reference to avoid extra docs maintenance work